certified-connectors/ShareEffect/apiDefinition.swagger.json

```diff
@@ -1,402 +0,0 @@
-{
```

Comment:
@bmp02050 Please let us know the reason for deleting another connector files in your PR. It is not allowed to delete a connector for any partner..

Reply:
I didn't mean to delete them. I was removing them from the commit.

Reply:
I was also told to exclude them and try again

Reply:
@bmp02050 Seems you need to update your fork with 'dev' repo. If we approve and merge your PR, the "ShareEffect" connector would be removed from the repo.

Reply:
I pulled the DEV and rebased it to my version. The ShareEffect stuff is still in there. Not sure how to exclude it without it deleting.